modules/cnf-programming-the-n3000-operator.adoc

```diff
@@ -0,0 +1,343 @@
+// CNF-1498 Validate and Document Intel SRO and SRIOV FEC Operator
+// Module included in the following assemblies:
+//
+// *cnf-optimize-data-performance-n3000.adoc
+
+[id="cnf-programming-the-n3000-operator_{context}"]
+= Programming the OpenNESS Operator for Intel FPGA PAC N3000
+
+When the Intel FPGA PAC N3000 is programmed with a vRAN 5G bitstream, the hardware exposes the Intel FPGA PAC N3000 with a vRAN 5G bitstream. This bitstream exposes the Single Root I/O Virtualization (SR-IOV) virtual function (VF) devices used to accelerate the FEC in the vRAN workload.
```

Comment:
Please s/devices used to/devices that are used to/ so it reads a little more smoothly.